sympy/crypto/crypto.py

```diff
@@ -3324,3 +3325,138 @@ def decipher_bg(message, key):
         orig_msg += (m ^ b)
 
     return orig_msg
+
+################# Columnar Transposition Cipher #################
+
+def encipher_columnar(message, key):
+    """
+    Performs Regular Columnar Encryption on plaintext and returns ciphertext
+
+    Examples
+    ========
+
+    >> from sympy.crypto.crypto import encipher_columnar
+    >> message = "HELLO WORLD"
+    >> encipher_columnar(message,"SYM")
+    "LWLEHLODEORY"
+
+    Parameters
+    ==========
+
+    message : string, the message to be encrypted.
+    key : string, the key to be used for encryption.
+
+    Returns
+    =======
+
+    The Encrypted message.
+
+    References
+    ==========
+    https://en.wikipedia.org/wiki/Transposition_cipher#Columnar_transposition
+
+    """
+    message =  ''.join(alphabet for alphabet in message if alphabet.isalpha())
+    key =  ''.join(alphabet for alphabet in key if alphabet.isalpha())
+    row_length = len(key)
+    words = len(message)
+    col_length = -(-words//row_length)          # Ceil Function
+    remainder = words%row_length
+
+    for i in range(row_length - remainder):
```

Comment:
or `message += ''.join(choice(string.lowercase) for i in range(row_length - remainder))`

Reply:
But, I guess then it would take random alphabets only from the message. I thought it would be a bit more secure if we take random characters from A-Z. What do you suggest?

Reply:
string.lowercase is the a-z alphabet ;-)